x/token/event.go

```diff
@@ -51,7 +51,7 @@ func NewEventIssueToken(e EventIssue, grantee, to sdk.AccAddress, amount sdk.Int
 	new := sdk.NewEvent(eventType)
 	for key, value := range attributes {
 		attribute := sdk.NewAttribute(key.String(), value)
-		new.AppendAttributes(attribute)
+		new = new.AppendAttributes(attribute)
```

Comment:
The `new` variable conflicts with the 'builtin' function. 
https://pkg.go.dev/builtin#new

Reply:
I fixed them.
TODO: fix typo in the relevant commit title (name -> new).